lib/mongo/operation/write/command/update.rb

```diff
@@ -45,11 +46,13 @@ class Update
           #
           # @return [ Hash ] The selector describing this update operation.
           def selector
-            { update: coll_name,
-              updates: updates,
-              writeConcern: write_concern.options,
-              ordered: ordered?
+            cmd = {
+                update: coll_name,
+                updates: updates,
+                writeConcern: write_concern.options,
+                ordered: ordered?
             }
+            bypass_document_validation ? cmd.merge!('bypassDocumentValidation' => true) : cmd
```

Comment:
Is there a constant for the string 'bypassDocumentValidation' that we could use here so we aren't instantiating another String? If not, I would use a Symbol anyway, as it would be a String singelton.